src/overview/getting-help.md

```diff
@@ -6,6 +6,6 @@ If you're facing any issue with Platform.sh, you can submit a support ticket fro
 
 You can also open a support ticket directly from your [user account](https://accounts.platform.sh/support).
 
-You are more than welcome to hop-on to our [public Slack channel](https://chat.platform.sh/).
+You are more than welcome to hop-on to our [public Slack channel](https://chat.platform.sh/), or on our [Talkus](https://www.talkus.io/) integration where you can reach our team directly by clicking on the live chat button.
```

Comment:
Wait, isn't the team actively saying we don't want people doing that?  Talkus only allows one person to answer a question, which is less than helpful.